changelog.md

```diff
@@ -1,3 +1,95 @@
+### v3.0.0
+
+Version 3.0.0 of Marionette has arrived and contains many improvements over version
+2.x but also some API Changes. Below is a list of the changes made to each component.
+
+To help the community transition over we have released a v2 patch tool to assist
+the upgrade. [Marionette Patch Tool] (https://github.com/marionettejs/marionette-v3-compat)
+
+#### View
+* `LayoutView` + `ItemView` merge and rename to `View`.
+* `Marionette.View` -> `ViewMixin`
+* Add `LayoutView` shortcut methods
+* `isDestroyed` and `isRendered` need to be made private with a public accessor method.
+* Add `before:show` event to `ChildView` when added after show
+* Call `Backbone.View` with result of options (163188eeb8)
+* Make `CompositeView`'s `renderChildren` public.
+* Rename `childEvents` to `childViewEvents`.
+* View: set `isDestroyed` to false by default breaking
+* Remove passing view options as a function breaking
+* Rename `templateHelpers` to `templateContext`
+* Make sure `before:render` is triggered before emptying regions.
+* Regions are not attached directly to the layout. Use `getRegion` instead.
+* Allow `CompositeView` to attach to existing HTML with `template:false`
+* Add `hasRegion` for layouts
+* Make `getChildView` private
+* Enable passing `preventDestroy` to `region.empty`.
+* `LayoutView` should remove its element before destroying child regions. There's now an option to turn it on, but now it’s available by default. This helps remove all of the synchronous paints going up the tree.
+
+#### CollectionView
+* Accept `ChildView` as a function
+* Check if filter is set
+* `CollectionView.destroyChildren` now triggers `destroy:children`
+* ChildView now accepts a function instead of using `getChildView` which was removed
```

Comment:
`getChildView` has been removed.

Also, move this to just under the point about `childView` above